rmm/monitor/src/rmi/mod.rs

```diff
@@ -51,6 +51,32 @@ pub const EXIT_SERROR: u8 = 6;
 
 pub type RMI = &'static dyn Interface;
 
+pub struct MapProt(usize);
+
+impl From<usize> for MapProt {
+    fn from(prot: usize) -> Self {
+        Self(prot as usize)
+    }
+}
+
+impl MapProt {
+    pub fn new(data: usize) -> Self {
+        MapProt(data)
+    }
+    pub fn set_bit(&mut self, prot: u64) -> &mut Self {
+        self.0 |= 1 << prot;
+        self
+    }
```

Comment:
`Mutable Self Reference`를 return하지 않아도 될 것 같습니다.

```suggestion
    pub fn set_bit(&mut self, prot: u64)  {
        self.0 |= 1 << prot;
    }
```